extractorapp/src/deb/resources/etc/georchestra/extractorapp/extractorapp.properties

```diff
@@ -0,0 +1,34 @@
+
+host=georchestra.mydomain.org
+instance=georchestra
+language=en
+
+# The base url of the extractorapp as accessible from outside the intranet
+servletUrl=http://georchestra.mydomain.org/extractorapp
```

Comment:
since we already have a ```host```var (above), maybe the servlet path only (+ an eventual http scheme) is required here